src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +26,31 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        final int numberLetters = 5;
```

Comment:
better move this variables in the class and make them as constants

Reply:
OK.